documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -0,0 +1,210 @@
+
+Hence, this guide explains how to install updated packages in `/usr/local`, to avoid conflicting with the base packages.
+
+## Warning
+
+Note carefully the comment in the section [Install container kernel and rootfs](#install-container-kernel-and-rootfs). You MUST update the packages listed regularly to ensure your Clear Containers benefit from performance improvements, new features and most importantly CVE security fixes.
+
+## Install updated glib and json-glib
+
+Note: these packages need to be built with the default RHEL compiler since they don't build by default with gcc 6.
+
+```
+$ sudo yum group install "Development Tools"
+$ sudo yum install zlib-devel libffi-devel
+$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
+$ curl -L -O "https://download.gnome.org/sources/glib/2.46/glib-2.46.2.tar.xz"
+$ tar -xvf glib-2.46.2.tar.xz
+$ pushd glib-2.46.2
+$ ./configure --prefix=/usr/local --disable-silent-rules && make -j5 && sudo make install
+$ popd
+
+$ curl -L -O "https://download.gnome.org/sources/json-glib/1.2/json-glib-1.2.2.tar.xz"
+$ tar -xvf json-glib-1.2.2.tar.xz
+$ pushd json-glib-1.2.2
+$ ./configure --prefix=/usr/local --disable-silent-rules && make -j5 && sudo make install
+$ popd
+
+```
+
+## System Dependencies for updated gcc
+This is required for gcc.
+
+```
+$ curl -L -O ftp://gcc.gnu.org/pub/gcc/infrastructure/mpc-1.0.3.tar.gz
+$ tar -xvf mpc-1.0.3.tar.gz
+$ pushd mpc-1.0.3
+$ ./configure
+$ make && sudo make install
+$ popd
+
+$ curl -L -O ftp://gcc.gnu.org/pub/gcc/infrastructure/gmp-6.1.0.tar.bz2
+$ tar -xvf gmp-6.1.0.tar.bz2
+$ pushd gmp-6.1.0
+$ ./configue
+$ make && sudo make install
+$ popd
+
+$ curl -L -O ftp://gcc.gnu.org/pub/gcc/infrastructure/mpfr-3.1.4.tar.bz2
+$ tar -xvf mpfr-3.1.4.tar.bz2
+$ pushd mpfr-3.1.4
+$ ./configure
+$ make && sudo make install
+$ popd
+
+```
+
+## Install updated gcc
+
+This is required for `qemu-lite`.
+
+```
+$ sudo yum install gmp-devel mpfr-devel libmpc-devel gcc-c++
+$ curl -L -O http://www.mirrorservice.org/sites/ftp.gnu.org/gnu/gcc/gcc-6.2.0/gcc-6.2.0.tar.gz
+$ tar xvf gcc-6.2.0.tar.gz
+$ pushd gcc-6.2.0
+$ ./configure --enable-languages=c --disable-multilib --prefix=/usr/local/gcc-6.2.0
+$ make -j5 && sudo make install
+$ popd
+
+```
+
+## Install qemu-lite
+
+Although this package installs the `qemu-lite` version of `qemu`, the binary will still be called `qemu-system-x86_64`.
+Requires at least gcc 5.
+
+```
+$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
+$ export PATH=/usr/local/gcc-6.2.0/bin:$PATH
+$ sudo yum install libcap-ng-devel pixman-devel libattr-devel libcap-devel
+$ curl -L -O https://github.com/01org/qemu-lite/archive/3f889234fcd31ac4119aae57559b1286e09e1568.tar.gz
+$ tar xvf 3f889234fcd31ac4119aae57559b1286e09e1568.tar.gz
+$ pushd qemu-lite-3f889234fcd31ac4119aae57559b1286e09e1568
+$ CC=/usr/local/gcc-6.2.0/bin/gcc ./configure \
+                                --disable-bluez         --disable-brlapi \
```

Comment:
I like the config option grouping here to avoid a huge long list, but could you reduce the amount of space used for the indent as currently it's huge! :smile: